### PR TITLE
wireshark plugin: update

### PR DESCRIPTION
**Pull request by coot**
* updated KeepAlive protocol
* updated TxSubmission protocol (v2)
* added Handshake protocol

### CODE REVIEW

File: ouroboros-network/wireshark-plugin/ouroboros_network.lua [hunk 1]
```diff
@@ -56,7 +63,8 @@ local txsubmission_msg_codes = {
 	[1] = "MsgReplyTxIds",
 	[2] = "MsgRequestTxs",
 	[3] = "MsgReplyTxs",
-	[4] = "MsgDone"
+	[4] = "MsgDone",
+        [6] = "MsgHello"
```
Comment (karknu) [inline ouroboros-network/wireshark-plugin/ouroboros_network.lua:67]: Indentation.

File: ouroboros-network/wireshark-plugin/ouroboros_network.lua [hunk 2]
```diff
@@ -76,6 +84,7 @@ ouroboros.fields = {
 	on_transmission_time,
 	on_conversation,
 	on_length,
+        on_handshake_msg,
```
Comment (karknu) [inline ouroboros-network/wireshark-plugin/ouroboros_network.lua:87]: Indentation.

File: ouroboros-network/wireshark-plugin/ouroboros_network.lua [hunk 3]
```diff
@@ -126,7 +135,7 @@ dissectOuroboros = function (tvbuf, pktinfo, root, offset)
 	local conv = tvbuf:range(offset + 4, 2)
 	local convId = conv:uint()
 
-	if (convId > 0x8000) then convId = convId - 0x8000
+	if (convId >= 0x8000) then convId = convId - 0x8000
```
Comment (karknu) [inline ouroboros-network/wireshark-plugin/ouroboros_network.lua:138]: 👍

**Review by karknu — APPROVED** [review]
LGTM

File: ouroboros-network/wireshark-plugin/ouroboros_network.lua (hunk 1 above)
Comment (coot) [reply ouroboros-network/wireshark-plugin/ouroboros_network.lua:67]: It seems this is caused by the endless war of `tabs` vs `spaces`.  I'd prefer to get rid of `tabs`, is that alright?

File: ouroboros-network/wireshark-plugin/ouroboros_network.lua (hunk 2 above)
Comment (coot) [reply ouroboros-network/wireshark-plugin/ouroboros_network.lua:87]: As above

**Comment by coot** [comment]
bors merge

**Pull request merged**